build-parent/pom.xml

```diff
@@ -404,6 +404,13 @@
         <version>${version.microprofile-jwt-auth}</version>
       </dependency>
 
+      <!-- SmallRye -->
+
+      <dependency>
+        <groupId>io.smallrye</groupId>
+        <artifactId>smallrye-metrics</artifactId>
+        <version>${version.smallrye.metrics}</version>
```

Comment:
Missing a closing dependency tag, causing the build to fail

Reply:
thanks! Seems to be a side effect of updating the branch :|